src/js/selection.js

```diff
@@ -1,18 +1,15 @@
-/*global Util */
-var Selection;
-
 (function () {
     'use strict';
 
     function filterOnlyParentElements(node) {
-        if (Util.isBlockContainer(node)) {
+        if (MediumEditor.util.isBlockContainer(node)) {
             return NodeFilter.FILTER_ACCEPT;
         } else {
             return NodeFilter.FILTER_SKIP;
         }
     }
 
-    Selection = {
+    var Selection = {
```

Comment:
Ok yeah, that's a fair point and it's a pretty standard pattern. I'll make that change.